src/cli/run_command.zig

```diff
@@ -1397,17 +1397,17 @@ pub const RunCommand = struct {
 
         // setup
 
+        if (!ctx.debug.loaded_bunfig) {
+            bun.CLI.Arguments.loadConfigPath(ctx.allocator, true, "bunfig.toml", ctx, .RunCommand) catch {};
+        }
```

Comment:
I think this can be moved above the fast run code and in `fn maybeOpenWIthBunJS`, remove the part where it loads bunfig.toml

Reply:
Fair, but I think there is some more checks one there like if its just a file to run, and we don't really need to load `bunfig` for them. I could also load it in main cli part like before PR too, but people seem to just really want it for running pkg scripts.

Reply:
bunfig is currently unconditionally loaded in maybeOpenWithBunJS, so it should have no impact

Reply:
Ah ok my bad, i didn't look close enough at the code. Fixed!